exp/util/util.go

```diff
@@ -0,0 +1,38 @@
+package util
```

Comment:
missing license and copyright boilerplate here.

Reply:
caught myself - need to get into the habit of `make verify` before pushing 🤦